rfcs/react-components/non-core-components.md

```diff
@@ -0,0 +1,388 @@
+- 👍 Other examples in OSS - Material UI/React all recommend peer dependencies
+- 👍 Easy for users to consume as long as core does not break
+- 👍 We test how 'complete' our public API is
+
+#### Cons
+
+- 👎 V10 upgrade could be unpleasant - need to check what range is supported for each package
+- 👎 More of our internal API becomes public
+
+### Documentation
+
+Fluent UI production components should always be documented regardless of whether they are core/non-core components.
+This section describes the ways to document non-core components.
+
+#### Current docsite
+
+Create a new section in our docsite explicitly for non-core components alongside core components.
+
+##### Pros
+
+- 👍 Existing infrastructure already setup
+- 👍 Enforces consistent documentation approach
+
+##### Cons
+
+- 👎 Do we want to document non-core components in the same way?
+- 👎 For separate repo, would need to think about how to consume stories
+- 👎 Needs work to support independently versioned packages
+
+#### New docsite
+
+Creates a new docsite for non-core components that will be linked to the current docsite.
+
+##### Pros
+
+- 👍 It's not hard to link between docsites that share the same look and feel
+- 👍 Clear boundary for users to distinguish core/non-core components
+- 👍 Possible to change documentation approach
+
+##### Cons
+
+- 👎 Manage the infra for another domain and publish
+- 👎 Needs work to support independently versioned packages
+
+### Parnter ownership and collaboration
+
+Partner teams might want to build scenario/product specific components, with guidance and support from the Fluent UI
+team. The Fluent UI team should have a say in the quality of the code that is shipped by partner teams under the
+Fluent UI umbrella and work with them to establish these quality criteria and ownership. This section proposes
+two different ways we should host partner code.
+
+#### GitHub
+
+##### Pros
+
+- 👍 Get different organizations in microsoft directly involved in open source
+
+##### Cons
+
+- 👎 Access management and compliance boundaries will be complicated
+- 👎 Figure out how to make sure partners maintain their projects properly
+
+#### ADO mirroring
+
+Partner teams can continue to host their code in respective ADO reponsitories and we host a mirror for their
+code in GitHub so that it is visible in open source for issue reporting and third party contributions.
+
+##### Pros
+
+- 👍 Already done in open source - React does this
+- 👍 Partners can stick to the tools they know but still have open source presence
+
+##### Cons
+
+- 👎 Complicated infra work
+- 👎 Potentially less accountability for partner teams that do this
+
+#### Both
+
+We let partner teams decide how they would like to participate in creating official Fluent powere components.
+
+##### Pros
+
+- 👍 Everybody is happy and can have a presence in open source
+- 👍 Maximum microsoft coverage for open source
+- 👎 Figure out how to make sure partners maintain their projects properly
```

Comment:
This is already as part of the cons and should probably be removed from the list of pros

```suggestion
```